website/docs/d/schematics_agent.html.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_schematics_agent"
+description: |-
+  Get information about schematics_agent
+subcategory: "Schematics Service API"
```

Comment:
existing resources has subcategory as Schematics
Please change `Schematics Service API` to `Schematics` in all docs

Reply:
done